Prace_domowe/Praca_domowa2/kinga_ulasik/PD2_Kinga_Ulasik_poprawione.ipynb

```diff
@@ -0,0 +1,1820 @@
+{
```

Comment:
<p>wymaga weryfikacji ponieważ może powodować overfitting - to jest plus?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>